target/ppc/fpu_helper.c

```diff
@@ -225,13 +225,41 @@ static void finish_invalid_op_arith(CPUPPCState *env, int op,
     finish_invalid_op_excp(env, op, retaddr);
 }
 
+static void finish_invalid_op_arith_defer(CPUPPCState *env, int op,
+                                          bool set_fpcc)
+{
+    CPUState *cs;
+
+    /* Update the floating-point invalid operation summary */
+    env->fpscr |= FP_VX;
+    /* Update the floating-point exception summary */
+    env->fpscr |= FP_FX;
+    if (env->fpscr & FP_VE) {
```

Comment:
This is wrong, the original (in `finish_invalid_op_arith`) is `fpscr_ve == 0` which is the opposite of what I wrote here. Will fix

Reply:
> This is wrong, the original (in `finish_invalid_op_arith`) is `fpscr_ve == 0` which is the opposite of what I wrote here. Will fix

Fixed in [fdf5092](https://github.com/PPC64/qemu/pull/81/commits/fdf50927b8cb69d31e91442971ec2385f56da916)